src/messaging/tests/TestExchangeMgr.cpp

```diff
@@ -46,13 +47,10 @@ using namespace chip::Inet;
 using namespace chip::Transport;
 using namespace chip::Messaging;
 
-using TestContext = chip::Test::IOContext;
+using TestContext = chip::Test::MessagingContext;
```

Comment:
Is the extra indirection needed? Why not:

```
chip::Test::MessagingContext sContext;
```

Reply:
The type is used to convert the `void * context` into the actual type. I'm not sure why not use the global context directly, but all test cases are written this way.